common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.java

```diff
@@ -29,14 +29,63 @@
 
 import com.microsoft.identity.common.exception.ClientException;
 
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_DEBUG_SIGNATURE;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_RELEASE_SIGNATURE;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.BROKER_HOST_APP_PACKAGE_NAME;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.BROKER_HOST_APP_SIGNATURE;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME;
+import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.COMPANY_PORTAL_APP_RELEASE_SIGNATURE;
+
 /**
  * Represents packageName and SignatureHash of a broker app.
  */
 public class BrokerData {
+
+    public static final BrokerData MICROSOFT_AUTHENTICATOR_DEBUG = new BrokerData(
+            AZURE_AUTHENTICATOR_APP_PACKAGE_NAME,
+            AZURE_AUTHENTICATOR_APP_DEBUG_SIGNATURE
+    );
+
+    public static final BrokerData MICROSOFT_AUTHENTICATOR_PROD = new BrokerData(
+            AZURE_AUTHENTICATOR_APP_PACKAGE_NAME,
+            AZURE_AUTHENTICATOR_APP_RELEASE_SIGNATURE
+    );
+
+    public static final BrokerData COMPANY_PORTAL = new BrokerData(
+            COMPANY_PORTAL_APP_PACKAGE_NAME,
+            COMPANY_PORTAL_APP_RELEASE_SIGNATURE
+    );
+
+    public static final BrokerData BROKER_HOST = new BrokerData(
+            BROKER_HOST_APP_PACKAGE_NAME,
+            BROKER_HOST_APP_SIGNATURE
+    );
+
+    private static final Set<BrokerData> debugBrokers = new HashSet<BrokerData>() {{
```

Comment:
If this is a constant, maybe DEBUG_BROKERS, and make it immutable by wrapping it in a Collections.immutableSet

Reply:
Addressed